iterminator/iterminator.py

```diff
@@ -41,6 +25,13 @@ def __init__(self, path):
     def name(self):
         return os.path.basename(self.path).split('.')[0]
 
+    @property
+    def scheme(self):
+        return parse_scheme(self.name)
+
+    def is_light(self):
+        return light_or_dark(self.name)
```

Comment:
This is reading from disk every time we access the `.is_light()` method. In practice it may not be noticeable, but it would probably be nicer to read the Plist data just once in `Scheme.__init__()` and store it on the instance as `self.data` or something.

That's making me think that, with these changes, it might make sense to combine the new code with the `Scheme` class in a new module `scheme.py`. Then `scheme.background_color` and `scheme.is_light` can be properties/methods accessing `self.data` rather than functions needing to reread the data on disk each time.